src/nvim/ui.c

```diff
@@ -536,7 +536,27 @@ void ui_flush(void)
   if (!ui_active()) {
     return;
   }
+
+  static bool was_busy = false;
+
   cmdline_ui_flush();
+
+  if (State != MODE_CMDLINE && curwin->w_floating) {
+    if (curwin->w_config.hide) {
```

Comment:
Why don't you merge these two conditions?

Reply:
mistake sry.